modules/schema/src/test/java/org/apache/ignite/internal/schema/catalog/CatalogToSchemaDescriptorConverterTest.java

```diff
@@ -58,8 +56,13 @@ public class CatalogToSchemaDescriptorConverterTest extends AbstractSchemaConver
 
     private static final int TEST_SCALE = 5;
 
+    private static Stream<NativeTypeSpec> nativeTypeSpecs() {
+        return Stream.of(NativeTypeSpec.values())
+                .filter(t -> t != NativeTypeSpec.NUMBER && t != NativeTypeSpec.BITMASK);
+    }
+
     @ParameterizedTest
-    @EnumSource(NativeTypeSpec.class)
+    @MethodSource("nativeTypeSpecs")
```

Comment:
from my point of view all similar changes in tests must be covered with TODO "issue to remove native types"